decidim-assemblies/app/permissions/decidim/assemblies/permissions.rb

```diff
@@ -33,6 +33,8 @@ def permissions
         user_can_list_assembly_list?
         user_can_read_current_assembly?
         user_can_create_assembly?
+        user_can_export_assembly?
```

Comment:
Can you add specs for these two actions (export and copy)? Implementing the `it_behaves_like` would be enough for now.